tldw_Server_API/app/core/DB_Management/ManuscriptDB.py

```
@@ -813,3 +813,842 @@ def reorder_items(
+            cur = conn.execute(
+                "UPDATE manuscript_citations "
+                "SET deleted = 1, last_modified = ?, version = ?, client_id = ? "
+                "WHERE id = ? AND version = ? AND deleted = 0",
+                (now, next_version, self._client_id, citation_id, expected_version),
+            )
+            if cur.rowcount == 0:
+                raise ConflictError(
+                    f"Citation {citation_id!r} delete failed (version conflict or not found).",
+                    entity="manuscript_citations",
+                    entity_id=citation_id,
+                )
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)